tests/bluetooth/bsim_bt/bsim_test_app/prj.conf

```diff
@@ -13,5 +13,3 @@ CONFIG_BT_L2CAP_DYNAMIC_CHANNEL=y
 CONFIG_BT_TINYCRYPT_ECC=y
 CONFIG_BT_DEVICE_NAME="bsim_test"
 CONFIG_BT_L2CAP_TX_BUF_COUNT=6
-
-CONFIG_BT_LL_SW_LEGACY=y
```

Comment:
@aescolar will this work?

Reply:
I will remove this file

Reply:
Yes, we just need to stop using it in CI (meaning remove the "app=tests/bluetooth/bsim_bt/bsim_test_app compile" line in compile.sh

Reply:
Thanks, done.